src/Service.Host/index.html

```diff
@@ -10,7 +10,7 @@
     <script type="text/javascript">
         window.APPLICATION_SETTINGS = {
             authorityUri: 'https://www-sys.linn.co.uk/auth/',
-            appRoot: 'http://localhost:61798',
+            appRoot: 'http://localhost:51698',
```

Comment:
Do we just pick a number for these that's different from the rest of the solutions?